src/client/l10n/phoneNumbers.tsx

```diff
@@ -0,0 +1,19 @@
+import { MarketLabel } from './locales'
+
+type PhoneNumbers = Partial<Record<MarketLabel, PhoneNumberData>>
+
+export type PhoneNumberData = {
+  displayNumber: string
+  hrefNumber: string
+  opens: string
+  closes: string
```

Comment:
Should we name these as `opensAt` and `closesAt` to hint that they refer to a time?

Reply:
Sounds good! :))